api/client.go

```diff
@@ -204,11 +204,6 @@ func (c Client) DeleteUser(id uid.ID) error {
 	return delete(c, fmt.Sprintf("/api/users/%s", id), Query{})
 }
 
-// Deprecated: use ListGrants
```

Comment:
When you merge this you should add an exclamation to the commit tag so that it bumps the next version we release as a breaking change.
>BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with [MAJOR](http://semver.org/#summary) in Semantic Versioning). A BREAKING CHANGE can be part of commits of any type.